### PR TITLE
[Part] Improve Part.Shape methods python documentation

### PR DESCRIPTION
**Pull request by FlachyJoe**
Format UserDocu to follow the template:
```
 Description
 method(args) -> return
 --
 More infos
```

### CODE REVIEW

File: src/Mod/Part/App/TopoShapePy.xml [hunk 1]
```diff
@@ -30,74 +30,98 @@ Sub-elements such as vertices, edges or faces are accessible as:
...
+        </UserDocu>
       </Documentation>
     </Methode>
     <Methode Name="exportBrepToString" Const="true">
       <Documentation>
-        <UserDocu>Export the content of this shape to a string in BREP format. BREP is a CasCade native format.</UserDocu>
+        <UserDocu>Export the content of this shape to a string in BREP format.
+exportBrepToString() -> string
+--
+BREP is a CasCade native format.</UserDocu>
```
Comment (luzpaz) [inline src/Mod/Part/App/TopoShapePy.xml:78]: `CasCade` ? Why not `OpenCasCade`

**Comment by luzpaz** [comment]
Will these also get translated ?

File: src/Mod/Part/App/TopoShapePy.xml (hunk 1 above)
Comment (FlachyJoe) [reply src/Mod/Part/App/TopoShapePy.xml:78]: Historical reason I suppose.

**Comment by FlachyJoe** [comment]
> Will these also get translated ?

I can't find any translated python embedded doc.
This file generate .h and .cpp at build time and I think we don't want localized build so lets keep it English only.

**Comment by FlachyJoe** [comment]
Please keep this open, I'll format every TopoShape*Py.xml

Does it need [skip ci] ?

**Pull request merged**